@xen-orchestra/proxy/docs/api.md

```diff
@@ -35,7 +35,9 @@ Content-Length: 69
 {"id":0,"jsonrpc":"2.0","method":"task.getAll","params":{"limit":10}}
 ```
 
-A result is a JSON-RPC response:
+A result can be:
+
+- JSON-RPC response
```

Comment:
This is not a valid list because the code parts are not in the list item.

Use alternative formatting here.